tests/test_job_management.py

```diff
@@ -543,6 +542,7 @@ def test_validation_job_id(dispatcher_live_fixture):
             'DEC': -29.74516667,
             'T1': '2017-03-06T13:26:48.000',
             'T2': '2017-03-06T15:32:27.000',
+            'T_format': 'isot'
```

Comment:
what happens if this is not passed?

Reply:
Now the `T_format` will be included for the calculation of the `job_id`, and it will be set to the default value of `isot`.

If not specified, or if it has a different value (eg `mjd`), it will be set to that default value, after having converted the value of the Time (eg `T1`, `T2`) to `isot` format.

Reply:
Ah ok, it the default functionality will take care of it.

Reply:
Precisely

Reply:
I see now it is not exactly the normal defaults mechanism since `T_format` does not correspond to a parameter name, but to it's units. It's a different point but related to the rest of the PR.